dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/passwordreset/PasswordResetServiceBean.java

```diff
@@ -39,6 +41,9 @@
     @EJB
     AuthenticationServiceBean authService;
 
+    @EJB
+    SettingsWrapper settingsWrapper;
```

Comment:
It should be private, we want to minimize access scope as much as possible. Comment about the class below.